src/rgw/rgw_rest_swift.cc

```diff
@@ -131,12 +131,30 @@ int RGWListBucket_ObjStore_SWIFT::get_params()
   return 0;
 }
 
+static void dump_container_metadata(struct req_state *, RGWBucketEnt&);
+
+void RGWListBucket_ObjStore_SWIFT::execute()
+{
+  RGWListBucket_ObjStore::execute();
```

Comment:
calls RGWListBucket::execute indirectly to avoid jumping over RGWListBucket_ObjStore::execute if it will defined in the future.